content/docs/300-on-brand-menu-api/300-api-keys.mdx

```diff
@@ -23,6 +23,8 @@ API keys can be created and managed from the Developer Portal’s [API keys](htt
 
    ie. `a98b440c16414f9c8a8c3fc239ce84fe`
 
+<Callout>**Single API Key vs Many API Keys: Although it's possible to create a unique API Key for each of your locations and menus, creating just a single API Key for all locations and menus simplifies API Key management and takes the guess-work out of remembering which key goes with which menus and locations.** </Callout>
```

Comment:
I don't think we should bold the entire statement but rather just the beginning: 

```suggestion
<Callout>**Single API Key vs Many API Keys**: Although it's possible to create a unique API Key for each of your locations and menus, creating just a single API Key for all locations and menus simplifies API Key management and takes the guess-work out of remembering which key goes with which menus and locations.</Callout>
```

Reply:
The callout is a bit big. What do you think about changing it to something like this:

**Single API Key vs Many API Keys**: You can create a single API key for multiple locations and menus to avoid remembering which key goes with which menus and locations.


<img width="658" alt="image" src="https://user-images.githubusercontent.com/1163252/180441197-b69cec2c-0fb1-450b-ae26-d6bd90325c49.png">

Reply:
@jschr agreed on bolding just the beginning. I was trying to be clear and knew it was getting a bit long, so was looking for suggestions, thanks for the update.

can you share the raw code so I can see what the font color and bold syntax looks like?

Reply:
The `**` wraps the text you want to bold: 

```
<Callout>**Single vs multiple API Keys**: You can create a single API key for multiple locations and menus to avoid remembering which key goes with which menus and locations.</Callout>
```

Reply:
ah, just an oversight, I put the closing in the wrong place